.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,59 +1,86 @@
 name: Bug/Crash Report
-description: Create a bug or crash report for Aliucord itself. To report bugs or crashes related to plugins, please go to our support server, or the plugin's repository and make an issue there instead.
+description: Report a bug or crash in Aliucord. For plugin issues, please use the plugin's repo or our support server.
 labels: [bug]
 body:
   - type: input
     id: discord
     attributes:
-      label: Discord Account
-      description: Who on Discord is making this request? Not required.
-      placeholder: username#0000
+      label: Discord username (Optional)
```

Comment:
I think we should remove this altogether, it's never been useful

Reply:
Nop, keep it, having context is better. I sometimes check if they sent something in support before.